File: app_manager/appmanager/src/main/java/app/editors/manager/mvp/views/login/EnterprisePhoneView.java
package app.editors.manager.mvp.views.login;

import moxy.viewstate.strategy.SkipStrategy;
import moxy.viewstate.strategy.StateStrategyType;

import app.editors.manager.mvp.views.base.BaseView;

@StateStrategyType(SkipStrategy.class)
public interface EnterprisePhoneView extends BaseView {
    void onSuccessChange();
}

File: app_manager/appmanager/src/main/java/app/editors/manager/mvp/views/login/EnterprisePhoneView.kt
package app.editors.manager.mvp.views.login

import app.editors.manager.mvp.views.base.BaseView
import moxy.viewstate.strategy.SkipStrategy
import moxy.viewstate.strategy.StateStrategyType

@StateStrategyType(SkipStrategy::class)
interface EnterprisePhoneView : BaseView {
    fun onSuccessChange(request: String)
}